docs/src/install-guide.md

```diff
@@ -1,32 +1,40 @@
-# Kani Installation Guide
+# Installation
 
 Kani must currently be built from source.
 
-In general, the following dependencies are required. Note: These dependencies may be installed by running the CI scripts shown below and there is no need to install them separately, for their respective OS.
+In general, the following dependencies are required.
 
-1. Cargo installed via rustup
+1. Cargo installed via [rustup](https://rustup.rs/)
 2. [CBMC](https://github.com/diffblue/cbmc) (>= 5.53.1)
 3. [CBMC Viewer](https://github.com/awslabs/aws-viewer-for-cbmc) (>= 2.10)
 
-## Installing on Ubuntu 20.04
+> **Note:** These dependencies may be installed by running the CI scripts shown
+> below and there is no need to install them separately, for their respective
+> OS.
```

Comment:
We moved this up originally because people would start installing things without having read it yet.